docker-compose.edx.yml

```
@@ -1,75 +1,75 @@
-version: '3.5'
+version: "3.5"
 
 services:
+  edx_mysql:
+    image: mysql:5.6
+    ports:
+      - "3316:3306"
+    env_file: env.d/edx
+    command:
+      mysqld --character-set-server=utf8 --collation-server=utf8_general_ci
```

```suggestion
    command: >
      mysqld --character-set-server=utf8 --collation-server=utf8_general_ci
```
Interesting, it work's also without the added `>` which replaces newlines with spaces in YAML. However, to follow the same "style" as `edx_lms` and `edx_cms` may be it's worth to add it in any case ?